client/CS130_Team13/Assets/Scripts/RemoteInfoClasses.cs

```diff
@@ -0,0 +1,30 @@
+namespace RemoteControllerNamespace
+{
```

Comment:
add this file for classes to prevent bloating of the RemoteController file. it got a bit hard to scroll around and it's easier to access this stuff when its in its own file